chrome-app/links.js

```diff
@@ -17,10 +17,13 @@ const ourRepos = ["repo:aws/jsii", "repo:aws/aws-cdk"];
 
 // LINKS
 const LINKS = [
-    { title: "Review", href: "https://github.com/pulls/review-requested" },
     { title: "Triage", href: gitHubQuery("https://github.com/issues", [...issue, ...ourRepos, "assignee:USERNAME", "label:bug", "-label:p0", "-label:p1", "-label:p2", "-label:p3"])},
+    { title: "P0", href: gitHubQuery("https://github.com/issues", [...issue, ...ourRepos, "assignee:USERNAME", "label:bug", 'label:p0'])},
     { title: "Shepherd", href: gitHubQuery("https://github.com/pulls", [...pr, "assignee:USERNAME", "-author:USERNAME" ])},
-    { title: "Finish", href: "https://github.com/pulls" },
-    { title: "Your bugs", href: gitHubQuery("https://github.com/issues", [...issue, ...ourRepos, "assignee:USERNAME", "label:bug"])},
-    { title: "Your issues", href: "https://github.com/issues/assigned" },
+    { title: "Author", href: "https://github.com/pulls" },
+    { title: "Review", href: "https://github.com/pulls/review-requested" },
```

Comment:
I put `Review` up front on purpose, because I experience a lot that my PRs go unreviewed for long stretches of time, and I wanted to encourage good behavior.

I think priority #1 is to unblock team mates, that's why I put it there.

Reply:
Okay, makes sense. We should discuss accountability around reviews. We have never agreed that if your review is requested, then you are responsible to review... Right now our process is a bit broken, so I am not sure *this* is going to fix it (because there is no process)

Reply:
Fixed